README.rst

```diff
@@ -3,5 +3,29 @@ ci_summit
 #########
 
 
+Description
+===========
 
-.. Add a brief (few sentence) description of what this package provides.
+``ci_summit`` provides test scripts to run the Rubin Observatory
+code on summit data.
+
+Test Data
+=========
+
+``ci_summit`` requires the test data in the ``testdata_ci_summit``
+package, which must be set up via eups first.
+
+Running Tests
+=============
+
+To run this package locally:
+
+#. Clone this package, `ci_builder <https://github.com/lsst-dm/ci_builder>`_, and
+`testdata_ci_summit <https://github.com/lsst-dm/testdata_ci_summit>`_.
+#. ``setup -r ci_builder``
```

Comment:
It doesn't need `scons`, and that's the cool thing about `ci_builder`!   `ci_imsim` runs that way too and I use it as the model for this repo. 

Trying to find docs and found [RFC-774](https://jira.lsstcorp.org/browse/RFC-774)

Or do you mean a basic scons file should be added?

Reply:
I meant that `ci_builder` needs to have `scons` run on it, since the end user will have a fresh clone of it.  Most people on the project would probably know to do so anyway, but it's always good if the documentation is explicit about things like that.

Reply:
Got it!  Thank you for clarifying!